src/product-type-import.js

```diff
@@ -118,6 +121,14 @@ export default class ProductTypeImport {
     if (isValid)
       return Promise.resolve()
 
-    return Promise.reject(validate.errors)
+    const productTypeName = productType.key || productType.name || 'unknown'
+    const error = new Error(
+      `Validation error on productType "${productTypeName}"`
```

Comment:
I think it'll be better to have more concise error messages. We know there's a validation error, but we don't know why or what's wrong

Reply:
I was thinking the same so I added [there](https://github.com/sphereio/sphere-product-type-import/pull/9/files/26fd2acd5b93ed9f85f707c9091cb43fd6ed5529#diff-bb1d9349429b29c9c1553e960b8660c4R127) the next line:
```js
+ ` - ${validate.errors[0].message}`
```

Which will add first error to the main message and then there is:
```js
error.body = {
  errors: validate.errors,
}
```

Which should add errors to the same place as where we have HTTP API errors .. (let me know if this is a good thing or if it should be directly in `error.errors`).